src/main.ts

```diff
@@ -162,7 +162,7 @@ async function run(): Promise<void> {
 
     # download coverage (owner flag doesn't work for download, prepend instead)
     if [ -n "${coverageOutput}" ]; then
-      ${cli} --verbosity ${verbosity} download ${owner}/$run -o ${coverageOutput};
+      ${cli} --verbosity ${verbosity} download  --owner ${owner} $run -o ${coverageOutput};
```

Comment:
This will break for customers on older version of Mayhem on single tenant, since the owner command would not be correctly supported. We can definitely risk that, but it could cause issues.

Reply:
I think the action downloads the latest cli from app and then runs that, why would it break for older versions on single tenant?